tests/remotes/base.py

```diff
@@ -6,10 +6,6 @@ class Base:
     def should_test():
         return True
 
-    @staticmethod
```

Comment:
`get_storagepath` was not used anywhere outside except for `Local`. I removed it for others and replaced with `_get_storagepath`.